### PR TITLE
ADEN-4319 Missing play params.

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [googleIma.js#L11](https://github.com/Wikia/app/blob/68033eb95f96bf6610dc91eb58500e987135a198/extensions/wikia/AdEngine/js/video/googleIma.js#L11): Function has 8 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/video/uapVideo.js [hunk 1]
```diff
@@ -75,7 +75,8 @@ define('ext.wikia.adEngine.video.uapVideo', [
 			});
 
 			params.videoTriggerElement.addEventListener('click', function () {
-				video.play();
+				var slotWidth = getSlotWidth(adSlot);
+				video.play(slotWidth, getVideoHeight(slotWidth, params));
```
Comment (drets) [inline extensions/wikia/AdEngine/js/video/uapVideo.js]: What a guys! :smile: @v3rth @fraszczakszymon

File: extensions/wikia/AdEngine/js/video/uapVideo.js [hunk 2]
```diff
@@ -75,7 +75,8 @@ define('ext.wikia.adEngine.video.uapVideo', [
 			});
 
 			params.videoTriggerElement.addEventListener('click', function () {
-				video.play();
```
Comment (drets) [inline extensions/wikia/AdEngine/js/video/uapVideo.js]: What a guys! :smile: @v3rth @fraszczakszymon

**Comment by drets** [comment]
After quick testing by @bulaszenko it seems it doesn't fix an issue :disappointed:  , so I'll move this ticket on top of @v3rth patch.

File: extensions/wikia/AdEngine/js/video/uapVideo.js (hunk 2 above)
Comment (drets) [inline extensions/wikia/AdEngine/js/video/uapVideo.js:78]: Sorry @v3rth @fraszczakszymon it seems I confused `play` and `playVideo`
(it's too bad to work from OS X, switching to NixOS back :smile: )

**Pull request closed without merging**